newIDE/app/src/UI/TreeView/index.js

```diff
@@ -235,7 +235,7 @@ const TreeView = <Item: ItemBaseAttributes>(
         !searchText ||
         forceAllOpened ||
         forceOpen ||
-        name.toLowerCase().includes(searchText) ||
```

Comment:
in which case do we have a React.Node? 
Does this mean in this case the search will not work?

Reply:
Only for the placeholders. The scene object placeholder is done through `i18n._(...)` and the search works for this.
But for the global object placeholder, since we want to have a link inside, I had to use `<Trans>...<Trans />`. That's the only case.

Reply:
ok then 👍